mods/cnc/rules/tech.yaml

```diff
@@ -1,6 +1,23 @@
+^TechBuildingHusk:
+	Inherits: ^CivBuildingHusk
+	Health:
+		HP: 1
+	DamageMultiplier:
+		Modifier: 0
+	Targetable:
+		RequiresForceFire: yes
+	Capturable:
```

Comment:
From a game design perspective, I think this is a bad idea.  Making dead tech structures repairable removes the incentive and requirement to defend them, and removes the strategy of killing enemy tech structures.